rotkehlchen/chain/evm/decoding/thegraph/decoder.py

```diff
@@ -58,6 +63,51 @@ def _decode_delegator_staking(self, context: DecoderContext) -> DecodingOutput:
             return self._decode_stake_locked(context)
         elif context.tx_log.topics[0] == TOPIC_STAKE_DELEGATED_WITHDRAWN:
             return self._decode_stake_withdrawn(context)
+        elif context.tx_log.topics[0] == DELEGATION_TRANSFERRED_TO_L2:
+            return self._decode_delegation_transferred_to_l2(context)
+        return DEFAULT_DECODING_OUTPUT
```

Comment:
space to denote end of if/elif and final return

```suggestion

        return DEFAULT_DECODING_OUTPUT
```